browser/css/jsdialogs.css

```diff
@@ -752,7 +752,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 }
 
 .ui-listbox-arrow {
-	content: '';
+	content: ' ';
```

Comment:
is that needed?

Reply:
Not really. I had some rendering issue at some point, the down arrow was not visible for the datetime field.
Undone.